src/screens/CreatePost/elements/CreatePollContainer/CreatePollContainer.ios.js

```diff
@@ -78,7 +85,9 @@
       <TouchableOpacity style={S.polldurationbutton} onPress={() => setIsDurationModalShown(true)}>
         <View style={S.row}>
           <Text style={S.fillparenttext}>Duration</Text>
-          <Text style={S.polldurationbuttontext}>{getDurationTimeText(selectedtime)}</Text>
+          <View style={S.polldurationbuttonview(isAnonym)}>
+            <Text style={S.polldurationbuttontext}>{getDurationTimeText(selectedtime)}</Text>
+          </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isAnonym` prop is being used to conditionally render the style of the `polldurationbuttonview`. This is a good practice as it allows for dynamic styling based on the state of the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->